content/logs/processing/parsing.md

```diff
@@ -114,6 +114,39 @@ Here is the list of all the matchers and filters natively implemented by Datadog
 {{% /tab %}}
 {{< /tabs >}}
 
+## Advanced Settings
+
+At the bottom of your grok processor tiles there is an advanced Setting section:
+
+{{< img src="logs/processing/parsing/advanced_settings.png" alt="Advanced Settings" responsive="true" style="width:80%;">}}
+
+* Use the **Extract from** field to apply your grok processor on a given attribute instead of the default `message` attribute.
+
+* Use the **Helper Rules** field to define tokens for your parsing rules. Helper rules helps you factorize grok patterns across your parsing rules.
```

Comment:
@l0k0ms I would mention that this is especially useful when you have several rules in the same grok parser that uses the same tokens.